modules/nf-core/kallisto/index/environment.yml

```diff
@@ -0,0 +1,6 @@
+channels:
```

Comment:
you'll need a conda environment name

Reply:
```suggestion
name: kallisto_index
channels:
```

Reply:
Ah, this was part of a bulk update, will bump modules